MODULE.bazel

```diff
@@ -0,0 +1,11 @@
+module(name = "org_eclipse_iceroyx", version = "2.95.0")
+
+bazel_dep(
+    name = "buildifier_prebuilt",
+    version = "7.1.2",
+    dev_dependency = True,
+)
+bazel_dep(name = "cpptoml", version = "0.1.1")
+bazel_dep(name = "googletest", version = "1.15.2")
```

Comment:
This is also just a dev dependency

Reply:
added at https://github.com/eclipse-iceoryx/iceoryx/pull/2338/commits/bd24a6d2b4762614ed2bdc3a0885c7c7b878b25c